meta-zarhus-distro/conf/distro/zarhus-distro-webkit.conf

```diff
@@ -1,37 +1,11 @@
-require conf/distro/poky.conf
+require conf/distro/include/zarhus-distro-common.conf
 
 DISTRO = "zarhus-distro-webkit"
-
-# distro name
 DISTRO_NAME = "Distro for Zarhus product with WebKit support"
-DISTRO_VERSION = "0.1.0"
-SDK_VENDOR = "-zarhussdk"
-
-MAINTAINER = "3mdeb Sp. z o. o. <contact@3mdeb.com>"
-
-TARGET_VENDOR = "-zarhus"
-
-# distro features
-# DISTRO_FEATURES_DEFAULT ?= "acl alsa argp bluetooth ext2 ipv4 ipv6 largefile pcmcia usbgadget usbhost wifi xattr nfs zeroconf pci 3g nfc x11 vfat"
-# POKY_DEFAULT_DISTRO_FEATURES = "largefile opengl ptest multiarch wayland vulkan"
-DISTRO_FEATURES = "${DISTRO_FEATURES_DEFAULT} systemd usrmerge"
-
-# explicitly use systemd and disable sysvinit completely
-VIRTUAL-RUNTIME_init_manager = "systemd"
-VIRTUAL-RUNTIME_initscripts = ""
-VIRTUAL-RUNTIME_syslog = ""
-DISTRO_FEATURES_BACKFILL_CONSIDERED += "sysvinit pulseaudio"
-
-ROOT_PASSWORD = "OZuZFzdcWUJMls5k5u7ErPhRvMQTHh0V"
-# According to https://docs.yoctoproject.org/singleindex.html#extrausers-bbclass
-# mkpasswd should be used in order to set account password, this password for
-# root was created with the meta-zarhus layer using the following command:
-# `printf "%q" $(mkpasswd -m sha256crypt $ROOT_PASSWORD)`
-# TODO: add way to automate generating ROOT_PASSWD, maybe split this before vs
-# after honister update, because the method changed
-ROOT_PASSWD = "TODO"
 
+# We can add suffix to main version if only Webkit distro changes e.g.
+# 0.1.0-1.0.0.
+DISTRO_VERSION:append = ""
```

Comment:
This one should follow release process https://github.com/zarhus/docs/pull/5, and I do not see anything related there. Actually, IIUC, the release process assumes we have a one distro for a repo, and do not cover multiple distros. Should we make different releases for different distros in one repo?

Reply:
Probably best option would be to just have one version for all distros even if only Webkit part changed.

Reply:
So, lets define `DISTRO_VERSION` inside `zarhus-distro-common.conf`.

Reply:
https://github.com/zarhus/meta-zarhus/pull/34/commits/d751e27a64a8939654ed89f1abd1b79ecc869214